### ISSUE TITLE
Changing Connection when creating new batch does not affect available mappers and messes up Config

### ISSUE THREAD

**Issue by kspurgin** [collectionspace/cspace-batch-import#39]
I log into the Heroku app as superuser

I begin to create a new batch and see the following:
![Screen Shot 2021-04-12 at 15 17 17](https://user-images.githubusercontent.com/450570/114451182-f60e3480-9ba4-11eb-9c46-a7e571184202.png)

I change Connection from "Core Dev Server" to "Fcart Dev Server" 

I still only see the core Mappers

The formatting of the Config box goes away.
![Screen Shot 2021-04-12 at 15 17 47](https://user-images.githubusercontent.com/450570/114451315-16d68a00-9ba5-11eb-8143-6f5d16e23e43.png)

Switching the Connection back to "Core Dev Server" does not restore the formatting of the Config box

**Comment by kspurgin** [comment]
If I select Fcart Dev Server and one of the core mappers, and then click submit, nothing appears to happen. My batch isn't created, but I also am not shown an error.

**Comment by mark-cooper** [comment]
1. Config losing formatting is resolved by: https://github.com/collectionspace/cspace-batch-import/commit/cfc93a23eb71d977918dc7310cc128b7ec4ead65
2. "Only see the core Mappers" was because the "fcart" connection had been updated to use core mappers. Presumably for testing.
3. "My batch isn't created, but I also am not shown an error." -- will need to return to this one. It shouldn't really be possible to select incompatible mappers because of the way the mappers are tied to the connection select (the core mappers were _correctly_ [even if misleadingly] associated with the "fcart" connection when I looked). However this behavior is tied into the Redis connection so we should enforce / verify the mappers are appropriate as part of batch validation (made a task for that).
